charts/prometheus/templates/prometheus-sidecar-configmap.yaml

```diff
@@ -0,0 +1,42 @@
+################################
+## Alertmanager sidecar ConfigMap
+#################################
+{{- if .Values.global.authSidecar.enabled }}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ template "prometheus.fullname" . }}-nginx-conf
+  labels:
+    tier: monitoring
+    component: {{ template "prometheus.name" . }}
+    chart: {{ template "prometheus.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  default.conf: |
+    upstream astro-prometheus {
+      server localhost:{{ .Values.ports.http }} ;
+    }
+    server {
+      server_name {{ template "prometheus.url" . }} ;
+      listen 8080  ;
+      listen [::]:8080  ;
+      location  = /auth {
+        proxy_set_header Host houston.{{ .Values.global.baseDomain }};
+        proxy_pass  https://houston.{{ .Values.global.baseDomain }}/v1/authorization;
+        {{ .Values.global.default_nginx_settings }}
+{{.Values.global.authSidecar.default_nginx_settings | indent 8  }}
+      }
+      location @401_auth_error {
+      	internal;
+      	add_header Set-Cookie $auth_cookie;
```

Comment:
@pgvishnuram can you please tweak formating? sounds like mixing spaces and tabs?

Reply:
I have a PR open that add pre-commit to CI, which has a hook to catch tabs. https://github.com/astronomer/astronomer/pull/1128

Reply:
I just merged that branch to master, then rebased this branch off master. Will run pre-commit on this branch, which will fix the problem.